test/images/metadata-concealment/README.md

```diff
@@ -0,0 +1,5 @@
+# metadata-concealment
+
+Runs various checks against the GCE metadata API. If the checks pass the job is run to completion. It fails and stops at the first error encountered.
```

Comment:
s/If the checks pass/If the checks pass,

Added an extra comma, so it's easier to understand.